### ISSUE TITLE
Importing a dir fails even if the dir contains an index.ts file

### ISSUE THREAD

**Issue by stacktracer** [mohd-akram/rollup-plugin-typescript#3]
Simple example at: [https://github.com/stacktracer/import-indexts-test](https://github.com/stacktracer/import-indexts-test)

I have a `.ts` file that tries to do `import "./<subdir>"`, where `<subdir>` contains an `index.ts` file. Both `vscode` and `tsc` are fine with it, but `rollup -c` fails with the following message:
```
Error: Could not resolve './hello' from src/index.ts
    at error (/home/mike/.npm/lib/node_modules/rollup/dist/rollup.js:3460:30)
    at /home/mike/.npm/lib/node_modules/rollup/dist/rollup.js:21850:25
```

Changing it to `import ./<subdir>/index` works ... but importing by subdir name would be nicer.

Not sure if the issue is with the typescript plugin or the resolver. Or if I just have something set up wrong.

**Comment by mohd-akram** [comment]
Thank you for reporting and for the example. This should be fixed in v1.1.2.

**Comment by stacktracer** [comment]
Works great in 1.1.2 -- thanks!
